### ISSUE TITLE
Test with gctorture and valgrind

### ISSUE THREAD

**Issue by krlmlr** [yihui/tikzDevice#85]
Look at code in old [GNUmakefile](https://github.com/yihui/tikzDevice/blob/8206a3bb04e792367d86adda69615701858806a9/GNUmakefile).

...and at `dpylr` tests.

**Comment by krlmlr** [comment]
After this and a revdep check, I guess the package is now ready for CRAN. @yihui: Is there anything else you'd like to see in this release?

**Comment by yihui** [comment]
No, I don't have anything else. Please feel free to make a new release. Thanks!

**Comment by krlmlr** [comment]
Now testing with

```
R -d "valgrind --tool=memcheck --leak-check=full --dsymutil=yes" --vanilla -e "devtools::test()" --args --use-gctorture
```
